tests/test_alchemy.py

```diff
@@ -287,11 +278,20 @@ class SQLAlchemySessionFactoryTestCase(unittest.TestCase):
     def test_create_get_session_from_sqlalchemy_session_factory(self):
         SessionGetterFactory.create()
         self.assertEqual(SessionGetterFactory._meta.sqlalchemy_session, models.session)
+        # Let's try again, since we now have a session set from the factory, and ensure we won't
+        # try to validate again
+        SessionGetterFactory.create()
 
     def test_create_raise_exception_sqlalchemy_session_factory_not_callable(self):
         message = "Provide either a sqlalchemy_session or a sqlalchemy_session_factory, not both"
```

Comment:
Can tighten the regexp.
```suggestion
        message = r"^Provide either a sqlalchemy_session or a sqlalchemy_session_factory, not both$"
```

Reply:
Done in [08cc710](https://github.com/FactoryBoy/factory_boy/pull/927/commits/08cc710fca63ba84d55d411a36d564a0f834f946)